### ISSUE TITLE
Guggy Integration cannot work

### ISSUE THREAD

**Issue by mehdiyahyaoui** [RocketChat/Rocket.Chat#5762]
Hello

**I'm Running 0.50.0
Snap
Ubuntu 64**

I created the web-hook with indication from Doc

But I have this error message in the Log:

`Exception in callback of async function: TypeError: Cannot read property 'statusCode' of null at /snap/rocketchat-server/446/programs/server/packages/rocketchat_integrations.js:1316:30 at runWithEnvironment (packages/meteor.js:1176:24)`

I set Trigger Word as "guggy" without ! mark
Basically nothing is happening when I type : `guggy "something"`
and when I go to log I see the message above

Help Plz

**Comment by graywolf336** [comment]
Can you try to update and try again? As this is working in the demo server :thinking:

**Comment by mehdiyahyaoui** [comment]
Hello 

Sorry for not reporting back on the issues 

My Problems is still the same - the integration not working

Here's what it shows on the log


```

 [34mI20170315-08:57:03.010(1) rocketchat_integrations rocketchat_integrations.js:854 [31mIntegrations ➔ Outgoing WebHook.error Error running Script in the Integration Guggy: 
[34mI20170315-08:57:03.013(1) rocketchat_integrations rocketchat_integrations.js:857 [31mIntegrations ➔ Outgoing WebHook.error Stack: 
[34mI20170315-08:57:03.014(1) rocketchat_integrations rocketchat_integrations.js:858 [31mIntegrations ➔ Outgoing WebHook.error   TypeError: Cannot read property 'gif' of undefined       at Script.process_outgoing_response (script.js:1:1188)       at evalmachine.<anonymous>:1:15       at ContextifyScript.Script.runInNewContext (vm.js:18:15)       at Object.exports.runInNewContext (vm.js:49:17)       at /snap/rocketchat-server/528/programs/server/packages/rocketchat_integrations.js:835:29       at RocketChatIntegrationHandler.executeScript (/snap/rocketchat-server/528/programs/server/packages/rocketchat_integrations.js:844:6)       at /snap/rocketchat-server/528/programs/server/packages/rocketchat_integrations.js:1462:32       at runWithEnvironment (packages/meteor.js:1176:24) 
[34mI20170315-08:57:03.017(1) rocketchat_integrations rocketchat_integrations.js:1496 [31mIntegrations ➔ Outgoing WebHook.error Error for the Integration "Guggy" to http://text2gif.guggy.com/guggify is: 
[34mI20170315-08:57:03.019(1) rocketchat_integrations rocketchat_integrations.js:1497 [31mIntegrations ➔ Outgoing WebHook.error { [Error: getaddrinfo EAI_AGAIN text2gif.guggy.com:80]   code: 'EAI_AGAIN',   errno: 'EAI_AGAIN',   syscall: 'getaddrinfo',   hostname: 'text2gif.guggy.com',   host: 'text2gif.guggy.com',   port: 80 }

below the version of my RC instance

![image](https://cloud.githubusercontent.com/assets/23338414/23938896/88692386-095e-11e7-90d7-be3c5adf4ca9.png)

```
